tests/bin/action_runner_test.py

```diff
@@ -146,3 +146,26 @@ def test_build_environment_too_long_run_id(self):
             TRON_RUN_NUM="10",
             TRON_ACTION="bar.baz",
         )
+
+
+class TestBuildLabels:
```

Comment:
suggestion: should we add some testcases where there's existing labels as well?

Reply:
Added them now 👍